superbench/analyzer/file_handler.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A module for data analysis."""
+
+from pathlib import Path
+import re
+
+import json
+import jsonlines
+import pandas as pd
+import yaml
+
+from superbench.common.utils import logger
+
+
+def read_raw_data(raw_data_path):
+    """Read raw data from raw_data_path and store them in self._raw_data_df.
+
+    Args:
+        raw_data_path (str): the path of raw data jsonl file
+    Returns:
+        DataFrame: raw data, node as index, metric name as columns
+    """
+    p = Path(raw_data_path)
+    raw_data_df = pd.DataFrame()
+    if not p.is_file():
+        logger.error('DataDiagnosis: invalid raw data path - {}'.format(raw_data_path))
+        return raw_data_df
+
+    try:
+        with p.open(encoding='utf-8') as f:
+            for single_node_summary in jsonlines.Reader(f):
+                raw_data_df = raw_data_df.append(single_node_summary, ignore_index=True)
+        raw_data_df = raw_data_df.rename(raw_data_df['node'])
```

Comment:
What does `rename` do?

Reply:
it will use node column as the index of the raw_data_df